### PR TITLE
Happy new year!

### PR DESCRIPTION
**Pull request by Abogical**
Should this step be automated?

### CODE REVIEW

**Comment by Partheo** [comment]
Happy new year!

File: src/borg/shellpattern.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def translate(pat, match_end=r"\Z"):
 
     This function is derived from the "fnmatch" module distributed with the Python standard library.
 
-    Copyright (C) 2001-2017 Python Software Foundation. All rights reserved.
+    Copyright (C) 2001-2018 Python Software Foundation. All rights reserved.
```
Comment (ThomasWaldmann) [inline src/borg/shellpattern.py]: maybe we should not update that one, it was copied in 2017 from the py stdlib.

Comment (Abogical) [reply src/borg/shellpattern.py]: Actually it was copied back in 2016 at #1989, but I modified that line last year for similar reasons. I'm sorry for this. Should we change it back to 2016?

Comment (ThomasWaldmann) [reply src/borg/shellpattern.py]: yes, maybe change it back.

**Comment by Abogical** [comment]
Reverted `shellpattern.sh` change.

**Comment by ThomasWaldmann** [comment]
@Abogical can you do same against 1.1-maint and 1.0-maint?

**Pull request merged**